ImageD11/sinograms/point_by_point.py

```diff
@@ -25,6 +25,12 @@
 
 import h5py
 import numba
+
+# ignore Numba dot product performance warnings?
+import warnings
+
+warnings.simplefilter('ignore', category=numba.core.errors.NumbaPerformanceWarning)
```

Comment:
Rather than suppressing the message, we should eventually fix the actual problem here